common/test/services/prismic/pages.test.js

```diff
@@ -1,7 +1,32 @@
 import { parsePage } from '../../../services/prismic/pages';
 import { pageWithoutBody } from '../../fixtures/pages/page';
+import { whatsOn } from '../../fixtures/pages/whats-on';
+import WhatsOnPage from '../../../../content/webapp/pages/whats-on';
+import { mountWithThemeAndContexts } from '../../../../common/test/fixtures/enzyme-helpers';
 
 describe('pages', () => {
+  describe('whats-on', () => {
+    it('renders different markup with/without a featured exhibition', () => {
```

Comment:
feels like a test of rendering the component which probably feels like the test should render in the component level

Reply:
I kept this out of the `/pages` directory, because Next is going to treat everything in there as if it were renderable, which isn't what we want. But maybe we should have a separate e.g. `/pages_test` directory for testing page rendering?

Reply:
I would be more tempted to have a `test` directory. It's pretty common across next projects, and other projects across the `wellcomecollection` org.

To be clear that would be in the project directory though, e.g. `content/webapp/test/pages/`.

Reply:
☝️  done this in latest commit

Reply:
I'm not sure I 100% get it.

I thought we were going to replicate the path structure within a `test` folder? This test is testing the `whats-on` page, should this exist in `content/webapp/test/pages/whats-on.test.tsx`?

Reply:
Ah, it looks like we have - but not removed this?

Reply:
ah, this file pre-existed this pr (for parsing, rather than rendering concerns)

Reply:
I really don't think we need both, I would stick with the most "close to user" type testing, which is the rending of the what's on page.

Reply:
Sooooorry – didn't realise that I'd not _actually_ removed the rendering tests from this file. Have done so now. There's a remaining test of `parseOnThisPage` – happy for that to stay?